stream-transformation-test/sample-transformations/pom.xml

```diff
@@ -23,5 +23,10 @@
             <artifactId>test-utils-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>uk.gov.justice.framework-api</groupId>
+            <artifactId>framework-api-core</artifactId>
+            <version>${framework-api.version}</version>
+        </dependency>
```

Comment:
If this is not a test dependency then it should be higher in the list

Reply:
done